cheta/get_telem.py

```diff
@@ -74,7 +74,7 @@ def sanitize_event_expression(expr):
                 ast.literal_eval(word)
                 tokens.append("LITERAL")
             except Exception:
-                raise ValueError("Cannot identify word {!r}".format(word))
+                raise ValueError("Cannot identify word {!r}".format(word)) from None
```

Comment:
Does the "from None" in this context change the output here?

Reply:
Yes, but it is what we want.